pkg/quarterdeck/middleware/ratelimiter.go

```diff
@@ -105,6 +105,10 @@ func (i *IPRateLimiter) cleanupIPInfo(ttl time.Duration) {
 }
 
 func RateLimiter(conf config.RateLimitConfig) gin.HandlerFunc {
+	if !conf.Enabled {
+		return nil
+	}
```

Comment:
This is the disabling feature; when `SetupRoutes` is called, it filters out any `nil` middleware, so by returning nil here, the middleware will not be added to Quarterdeck.